lib/quest/quest_watcher.rb

```diff
@@ -115,7 +109,7 @@ def start_watcher
       Quest::LOGGER.info("Initializing watcher watching for changes in #{quest_watch}")
       @watcher = FileWatcher.new(quest_watch)
       @watcher_thread = Thread.new(@watcher) do |watcher|
-        watcher.watch do |f|
+        watcher.watch do |changed_file_path|
           run_specs
```

Comment:
Ah ok, that makes sense.  Also I just noticed I was wrong about the other line :)